gdx/src/com/badlogic/gdx/math/Rectangle.java

```diff
@@ -195,6 +195,19 @@ public boolean contains (Vector2 point) {
 		return contains(point.x, point.y);
 	}
 
+	/** @param circle the circle
+	 * @return whether the circle is contained in the rectangle */
+	public boolean contains (Circle circle) {
+		float xmin = circle.x - circle.radius;
+		float xmax = circle.x + circle.radius;
+
+		float ymin = circle.y - circle.radius;
+		float ymax = circle.y + circle.radius;
+		
+		return (xmin >= x && xmin <= x + width) && (xmax >= x && xmax <= x + width)
+			&& (ymin >= y && ymin <= y + height) && (ymax >= y && ymax <= y + height);

```

Comment:
since `min <= max`, this can be reduced to: `xmin >= x && xmax <= x + width && ymin >= y && ymax <= y + height`